### PR TITLE
Allow on-deploy scripts to access /conf

### PR DESCRIPTION
**Pull request by paul-bjorkstrand**
Since `/conf` is the successor of `/etc`, on-deploy scripts should be able to modify things here as well. This is primarily to allow on-deploy scripts to update editable templates & template policies

### CODE REVIEW

**Comment by paul-bjorkstrand** [comment]
Updated PR to fix merge issues (related to #2015)

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,10 +6,15 @@ after the 3.9.0 release. All changes up until the 3.9.0 release can be found in
...
 
 ## [Unreleased]
+<!-- Keep this up to date! After a release, change the tag name to the latest release -->
+[Unreleased]: https://github.com/Adobe-Consulting-Services/acs-aem-commons/compare/acs-aem-commons-4.3.0...HEAD
 
 ### Fixed
 - #1975 - Split application content from mutable content
 
+### Added
+- #2017 - Added read/write access to `/conf` for acs-commons-on-deploy-scripts-service uwer
```
Comment (HitmanInWis) [inline CHANGELOG.md]: Latest `master` needs to be merged into this ticket so that this can then be put back up in the "Unreleased" section.

**Comment by HitmanInWis** [comment]
Outside of merging latest master in and updating the CHANGELOG, this is tested and good to go.

**Comment by adamcin** [comment]
Hey @paul-bjorkstrand or @klcodanr , can you also add `/conf/rep:policy` to the `check-expected-policy-paths` check in `ui.content/pom.xml` ?

**Comment by HitmanInWis** [comment]
> Hey @paul-bjorkstrand or @klcodanr , can you also add `/conf/rep:policy` to the `check-expected-policy-paths` check in `ui.content/pom.xml` ?
> 
> EDIT: add link to desired location:
> 
> https://github.com/Adobe-Consulting-Services/acs-aem-commons/blob/4c36b113c98e5106e966dfc5b86c8bd39177d057/ui.content/pom.xml#L111

@adamcin do you mean `<path>/conf/rep:policy</path>`?

**Comment by adamcin** [comment]
@HitmanInWis yes, that is what I mean.

**Comment by klcodanr** [comment]
Checking w/ Paul on our internal Slack. I can as well, but... lazy ;-)

**Comment by paul-bjorkstrand** [comment]
@HitmanInWis I added added 
```

File: CHANGELOG.md (hunk 1 above)
Comment (HitmanInWis) [reply CHANGELOG.md]: This comment in CHANGELOG still needs to be moved up to the "Unreleased" section.

Comment (paul-bjorkstrand) [reply CHANGELOG.md]: I thought it was handled already, done now.

**Pull request merged**